server/base/src/test/java/org/apache/accumulo/server/tablets/TabletTimeTest.java

```diff
@@ -33,6 +33,10 @@
 public class TabletTimeTest {
   private static final long TIME = 1234L;
   private MillisTime mtime;
+  private static final MetadataTime m1234 = new MetadataTime(1234, TimeType.MILLIS);
```

Comment:
Seems like object like these could be created in MetadataTimeTest and that would make the test shorter.

Reply:
agree, I was being lazy.

Reply:
Last commit includes MetadataTimeTest cleanup. :)

Reply:
> agree, I was being lazy.

I have written many hard to read test in the past.  Doing lots of code reviews has motivated me to spend more time making my test more readable.  My least favorite part of doing a code review is looking at the test, however I think a thorough review of the test pays future dividends so I do it.  When I saw how you made one test more readable I thought that would be nice in the other test.